src/main/java/org/dcsa/core/repository/ExtendedRepositoryImpl.java

```diff
@@ -90,9 +90,11 @@ private void handleColumn(Class<?> c, T object, DBEntityAnalysis<T> dbEntityAnal
                 throw new IllegalStateException("Internal error: Attempting to set a value without a backing field!?");
             }
             fieldType = combinedModelField.getType();
-            if (fieldType.isEnum() && value instanceof String) {
-                // Convert to an enum
-                value = parseEnum(fieldType, (String)value);
+            if (fieldType.isEnum()) {
+                if (value != null && value instanceof String) {
```

Comment:
```suggestion
                 if (value instanceof String) {
```

`instanceof` implies `value != null`

Reply:
already done...